docs/configuration.md

```diff
@@ -1,53 +1,109 @@
+
+> NOTE: these limits are based on each connection.
+
+## Prometheus
+
+| Name                                     | Description                                                  | Default |
+| ---------------------------------------- | ------------------------------------------------------------ | ------- |
+| kopPrometheusStatsLatencyRolloverSeconds | Kop metrics expose to prometheus rollover latency in seconds. | 60      |
+
+## Group Coordinator
+
+This sections lists configurations about group coordinator and `__consumer_offsets` topic that is used to store committed offsets.
 
 |Name|Description|Default|
 |---|---|---|
-|messagingProtocols|  Messaging protocols available for being loaded by Pulsar Broker |kafka|
-|kafkaListeners|Comma-separated list of URIs we will listen on and the listener names.<br>e.g. PLAINTEXT://localhost:9092,SSL://localhost:9093.<br>If hostname is not set, bind to the default interface.||
-|kafkaAdvertisedListeners|Listeners to publish to ZooKeeper for clients to use.<br>The format is the same as `kafkaListeners`.||
-|listeners|Deprecated - use `kafkaListeners` instead. ||
-|kafkaTenant| The default tenant of Kafka's topics |public|
-|kafkaNamespace| The default namespace of Kafka's topics |default|
-|kafkaMetadataTenant| Tenant used for storing Kafka metadata topics |public|
-|kafkaMetadataNamespace| Namespace used for storing Kafka metadata topics  |__kafka|
-|maxReadEntriesNum| Maximum number of entries that are read from cursor once per time  |5|
 |groupMinSessionTimeoutMs| The minimum allowed session timeout for registered consumers<br>Shorter timeouts result in quicker failure detection while require more frequent consumer heart beating, which can overwhelm broker resources.  |6000|
 |groupMaxSessionTimeoutMs| The maximum allowed session timeout for registered consumers. <br>Longer timeouts give consumers more time to process messages between heartbeats while require longer time to detect failures. |300000|
 |groupInitialRebalanceDelayMs| The time the group coordinator waits for more consumers to join a new group before performing the first rebalance <br> A longer delay potentially reduces rebalances, but increases the time until processing begins.  |3000|
-|offsetsTopicCompressionCodec| Compression codec for the offsets topic <br>compression may be used to achieve "atomic" commits  |N/A|
+|offsetsTopicCompressionCodec| Compression codec for the offsets topic <br>compression may be used to achieve "atomic" commits  ||
 |offsetMetadataMaxSize| The maximum size in bytes for a metadata entry associated with an offset commit  |4096|
 |offsetsRetentionMinutes| Offsets older than this retention period are discarded |4320|
 |offsetsMessageTTL| Offsets message ttl in seconds. | 259200 |
 |offsetsRetentionCheckIntervalMs| Frequency at which to check for stale offsets  |600000|
 |offsetsTopicNumPartitions| Number of partitions for the offsets topic  |8|
```

Comment:
Whether the `__consumer_offsets` topic partition default number keep with kafka ?

Reply:
fixed